### ISSUE TITLE
False positive for large_tuple

### ISSUE THREAD

**Issue by lightsprint09** [realm/SwiftLint#1328]
The following raises an large_tuple error:

```swift
 var callback: ((Data?, URLResponse?, Error?) -> Void)?
```

**Comment by marcelofabri** [comment]
Duplicate of #1257 and fixed in `master` by #1259.
